nodepool/files/nodepool.yml

```diff
@@ -127,6 +127,19 @@ providers:
         availability-zones: []
         labels: *provider_pools_main_labels_block
 
+  - name: phobos-cloud
+    region-name: "RegionOne"
+    cloud: phobos-cloud
+    boot-timeout: 120
+    hostname-format: nodepool-{label.name}-{provider.name}-{node.id}
+    image-name-format: nodepool-phobos-{image_name}-{timestamp}
+    diskimages: *provider_diskimage_block
+    pools:
+      - name: main
+        max-servers: 20
+        availability-zones: []
+        labels: *provider_pools_main_labels_block
```

Comment:
Is the ``key-name`` for phobos also called ``jenkins`` or something else? If something else, then the ``labels`` block will have to be adjusted to use the right name for phobos.

Reply:
According to https://rpc-openstack.atlassian.net/wiki/spaces/RE/pages/449314836/Phobos+OpenStack the key-name should be ``openstack_phobos_nodepool`` - so either we rename the key, or we have to duplicate a ton of content here. I think it may be best for us to just rename the key to keep the configuration simpler.

Also, I don't see a way for nodepool to use a different private key to do the authentication - so the key we use should be the same across Phobos and Public Cloud.